app/controllers/rooms_controller.rb

```diff
@@ -240,7 +242,16 @@ def validate_accepted_terms
 
   def validate_verified_email
     if current_user
-      redirect_to resend_path unless current_user.email_verified
+      redirect_to account_activation_path(current_user) unless current_user.email_verified
+    else
+      redirect_to root_path
```

Comment:
Why are we sending the user to root?, shouldn't they just stay where they were?

Reply:
In the case where someone (who is not signed in) attempts to make an http request (to any action other than show or join), I believe that they should be redirected to the root path. Maybe it would be better if we add a flash informing the user that they need to sign in before they access that functionality